src/cli.ts

```diff
@@ -11,21 +11,27 @@ import { createPostmanCollection } from "./index.ts";
 function help() {
   console.log(`Error: not enough arguments.
 Usage:
-	deno run index.ts <GRAPHQL_ENDPOINT_URL>
+  deno run index.ts <GRAPHQL_ENDPOINT_URL>
 Options:
   --out=OUTPUT_FILE  Output file path
   -H="header: value"  Header to add to the request, the flag can be used multiple times.
+  --Hapikey="header: value" API key authorization header.
 Help:
   deno run index.ts [--help | -h]
 `);
 }
 
-const args = parse(Deno.args, { boolean: ["help", "h"], collect: ["H"] }) as {
+const args = parse(Deno.args, {
+  boolean: ["help", "h"],
+  collect: ["H"],
+  string: ["Hapikey"],
```

Comment:
I think the flag name is not really explicit, something like '--Global-Header', that may be shortened to '-GH', would be more explicit. Or maybe `--AuthHeader`?